frontend/src/metabase/query_builder/components/notebook/lib/steps.unit.spec.ts

```diff
@@ -0,0 +1,228 @@
+import { createMockMetadata } from "__support__/metadata";
```

Comment:
This file has a lot of changes and has been converted to TS - that's why it's shown as a new file.